litmus-portal/backend/graphql-server/pkg/graphql/subscriptions/subscription.go

```diff
@@ -50,8 +50,8 @@ func SendWorkflowRequest(wfRequest *database.ChaosWorkFlowInput, r store.StateDa
 	}
 
 	r.Mutex.Lock()
-	if r.ClusterEventPublish != nil {
-		for _, observer := range r.ConnectedCluster {
+	for cluster_id, observer := range r.ConnectedCluster {
```

Comment:
This is not required, the ConnectedCluster is a map where the key is the cluster_id itself, so you can just directly do :
```
if observer,ok:=r.ConnectedCluster[wfRequest.ClusterId];ok{
  observer<-newAction
}
```

Reply:
You also need to handle the scenario where `ok` is false, i.e. the cluster is no more connected.

Reply:
Yes, Done